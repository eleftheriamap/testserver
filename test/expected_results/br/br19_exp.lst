
test/exp/test/gen/br/br19_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	d297325c 	mov	x28, #0xb992                	// #47506
   4:	f2a0017c 	movk	x28, #0xb, lsl #16
   8:	d2800014 	mov	x20, #0x0                   	// #0
   c:	f2b14014 	movk	x20, #0x8a00, lsl #16
  10:	f8000794 	str	x20, [x28], #0
  14:	d61f0380 	br	x28
  18:	8a000000 	and	x0, x0, x0
