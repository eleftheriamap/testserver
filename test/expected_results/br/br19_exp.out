Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = 0000000000000000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 000000008a000000
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 00000000000bb992
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 00000000000bb992
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0xd297325c
0x00000004 : 0xf2a0017c
0x00000008 : 0xd2800014
0x0000000c : 0xf2b14014
0x00000010 : 0xf8000794
0x00000014 : 0xd61f0380
0x00000018 : 0x8a000000
0x000bb994 : 0x00008a00
